src/OpenTelemetry.AutoInstrumentation/CallTarget/Handlers/IntegrationMapper.cs

```diff
@@ -524,6 +580,9 @@ internal static CreateAsyncEndMethodResult CreateAsyncEndMethodDelegate(Type int
          *      - TReturn OnAsyncMethodEnd<TTarget, TReturn>(TTarget instance, TReturn returnValue, Exception exception, CallTargetState state);
          *      - TReturn OnAsyncMethodEnd<TTarget, TReturn>(TReturn returnValue, Exception exception, CallTargetState state);
          *      - [Type] OnAsyncMethodEnd<TTarget>([Type] returnValue, Exception exception, CallTargetState state);
+         *      - TReturn OnAsyncMethodEnd<TTarget, TReturn>(TTarget instance, TReturn returnValue, Exception exception, in CallTargetState state);
```

Comment:
I think that you should go through all bytecode integrations and add `in` to the parameters.
As an example please check src\OpenTelemetry.AutoInstrumentation\Instrumentations\GraphQL\ExecuteAsyncIntegration.cs ->OnAsyncMethodEnd

Reply:
I'd like to do this in a follow-up PR, is that okay with you?

Reply:
Sure.

Reply:
#2424